tests/testthat/test-convert_rle_to_df.R

```diff
@@ -0,0 +1,63 @@
+prepared_unreliable_regions <- function() {
+  return(
+    combined_unreliable_regions %>%
```

Comment:
I'd be tempted to explicitly define a testing specific version of unreliable regions with two main goals in mind.

1. ensure that tests don't break if the we need to update unrelated_regions at a later date
2. Define specific regions so that it is easy to tell in testing if the unreliable regions have been applied. As is I can't tell at a glance from the snapshots if the unreliable regions are even doing anything